tests/timeseries/test_moving_window.py

```diff
@@ -14,6 +14,7 @@
 from frequenz.channels import Broadcast, Sender
 
 from frequenz.sdk.timeseries import Sample
+from frequenz.sdk.timeseries._base_types import UNIX_EPOCH
 from frequenz.sdk.timeseries._moving_window import MovingWindow
 from frequenz.sdk.timeseries._resampling import ResamplerConfig
```

Comment:
Test should try to use public imports when possible. I think all symbols in here are available simply in `frequenz.timeseries`.
```suggestion
from frequenz.sdk.timeseries import Sample, UNIX_EPOCH, MovingWindow, ResamplerConfig
```